docs/pages/build/building-from-github.mdx

```diff
@@ -6,10 +6,6 @@ description: Learn how to trigger builds on EAS for your app using the Expo GitH
 
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 
```

Comment:
I'd still expect us to have form of introduction on this page. Just a small description as to what info this page provides or what someone can expect will be sufficient.